book/website/community-handbook/bookdash/bookdash-events.md

```diff
@@ -0,0 +1,142 @@
+
+The day ends with a group dinner hosted by the core team to allow participants to relax after an intense discussion session and enjoy meals together in an informal setting.
+In between the meals, participants are invited to give lightning talks where they can share their personal stories and talk about a project that they’re either working on or are excited about.
+
+### Agenda for the Day-1
+
+```
+14:00-15:00 BST: Arrive at the book dash venue for informal networking
+15:00-16:00 BST: Intro to _The Turing Way_ and book dash
+17:00-17:30 BST: Brainstorming in groups
+17:30-18:00 BST: Report out
+18:30 BST: Arrive at the dinner venue
+19:00 BST: Dinner begins
+20:30-21:00 BST: Lightning talks*
+21:00-22:00 BST: Dinner continues with networking
+```
+
+## Day-2: coworking day
+
+The next day of the book dash is a collaboration and coworking day.
+This day is distributed into 4 sessions of 1 hour 15 minutes each, separated by 15 minutes reflection followed by a break.
+
+The first session facilitates discussions by bringing all the attendees in small groups with their collaborators based on their discussion from the previous day.
+The collaborating groups start working on their road map and plans for their contributions as chapters, subchapters or other types of contents.
+Participants are also encouraged to come up with the main sections of their contributions and collect their thoughts in bullet points on a GitHub issue.
+During the second session, participants work in self-organised groups or individually with the support of organisers and helpers.
+This session ends with a quick report out from each group followed by a lunch break.
+
+During the third and fourth session, participants come back to continue working uninterrupted on their contribution ideas and take break as they need.
+At the end of this session, participants gather all their contributions  on _The Turing Way_ GitHub repository (issues, pull requests and notes) and add that in the shared notes before taking a short break.
+
+The last session is the final report out and celebration of each other's work.
+The book dash closes with anonymous feedback on the {ref}`Pluses and Delta<ch-bookdash-feedback-template>` notes.
+
+### Agenda for Day-2
+
+```
+8:30 BST: Arrive at the dashing venue
+8:45 BST: Intro to the second day of the book dash
+9:00 BST: Coworking starts (session-1)
+10:15 BST: Report
+11:00 BST: Coworking continues (session-2)
+12:15 BST: Report out
+12:30 BST: Lunch break
+13:30 - 16:30 BST: Coworking continues (session-3 and 4)
+16:30 BST: Group share outs and celebrations
+17:00 BST: Event closing and feedback
+```
+
+## Creating an environment for self-paced contributions
+
+We acknowledge that everyone prefer different setting to work effectively and attentively.
+Therefore we create environment for self-paced contributions to _The Turing Way_ project.
+
+Some recommendation to achieve that is by managing space by:
```

Comment:
Suggest to change to either: 
"Some recommendations to achieve this by managing the space at the book dash by:" 
OR 
"Some recommendations to achieve this by managing space are:"